build-logic/build-platform/build.gradle.kts

```diff
@@ -73,5 +73,45 @@ dependencies {
         }
         api("net.bytebuddy:byte-buddy") { version { strictly("1.10.21") } }
         api("org.objenesis:objenesis") { version { strictly("3.1") } }
+        api("com.github.mwiede:jsch:0.2.0") {
+            because("`com.jcraft:jsch` is outdated - `com.github.mwiede:jsch` is a drop-in replacement")
+        }
+    }
+
+    components {
```

Comment:
None of the changes below are required. There is no real dependency resolution happening in a `java-platform` project.

Reply:
https://github.com/gradle/gradle/pull/20387#discussion_r861992460